app/assets/javascripts/host_edit.js

```diff
@@ -6,6 +6,20 @@ $(document).on('ContentLoad', function() {
 $(document).on('AddedClass', function(event, link) {
   load_puppet_class_parameters(link);
 });
+$(document)
+  .on('change', '.hostgroup-select', function(evt) {
+    hostgroup_changed(evt.target);
+  }).on('change', '.host-form-compute-resource-handle', function(evt) {
```

Comment:
What's the difference between `select` suffix and `handle` suffix?
Aren't they all selects?

Reply:
The handle class is used more than once :thinking:
I am terrible with naming... any ideas?

Reply:
What about something like this? (not sure if it's the right syntax)
```suggestion
  }).on('change', '.host-form-compute-resource-select, .host-form-compute-profile-select', function(evt) {
```